cloud/azure/bin/lib/bastion.sh

```diff
@@ -141,6 +141,36 @@ function bastion::get_pg_restore_command() {
       -a < ${3}"
 }
 
+#######################################
+# Get the command to dump db data.
+# Arguments:
+#   1: the postgres host to connect to
+#   2: the vaultname where secrets are stored
+#   3: path of the dump data
+#######################################
+function bastion::get_pg_dump_command() {
+  local DB_PASSWORD=$(bastion::get_pg_password "${2}")
+  echo "export DEBIAN_FRONTEND='noninteractive'; \
+    sudo sh -c 'echo \"deb http://apt.postgresql.org/pub/repos/apt \
+      \$(lsb_release -cs)-pgdg main\" > /etc/apt/sources.list.d/pgdg.list'; \
+    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc \
+      | sudo apt-key add -; \
+    yes | sudo apt-get update > /dev/null; \
+    yes | sudo apt-get install postgresql-14 > /dev/null; \
+    PGPASSWORD='${DB_PASSWORD}' /usr/lib/postgresql/14/bin/pg_dump \
+      -U psqladmin@${1} \
+      -w \
+      -Fc \
+      -h ${1} \
+      -t programs \
```

Comment:
Why are we specifying a list of tables?

Reply:
So that it doesn't export applicant data, PII

Reply:
I feel like that should be a flag.  Like maybe an admin wants to export the PII for a backup.  Or at least put it into the docstring to make this clear

Reply:
imo we should add that flag when someone needs it